war/.eslintignore

```diff
@@ -12,6 +12,4 @@ jsbundles
 # External scripts
 src/main/webapp/scripts/yui
 src/main/webapp/scripts/prototype.js
-src/main/webapp/scripts/*.js
 src/main/js/plugin-setup-wizard/bootstrap-detached.js
-jest.config.js
```

Comment:
Cleaning up after #6908 which removed `jest.config.js` but forgot to remove this exclusion.